src/main/kotlin/edu/cwru/raycaster/Raycaster.kt

```diff
@@ -1,32 +1,68 @@
 package edu.cwru.raycaster
 
-import javafx.animation.AnimationTimer
 import javafx.application.Application
+import javafx.scene.Cursor
 import javafx.scene.Scene
 import javafx.scene.control.Label
-import javafx.scene.image.PixelFormat
 import javafx.scene.input.KeyCode
 import javafx.scene.layout.HBox
 import javafx.scene.layout.VBox
 import javafx.scene.paint.Color
+import javafx.scene.robot.Robot
 import javafx.stage.Stage
-import java.nio.ByteBuffer
+import kotlinx.coroutines.*
 import kotlin.math.PI
 import kotlin.math.floor
 
-const val NS_IN_S = 1_000_000_000.0
+fun Double.format(scale: Int) = "%.${scale}f".format(this)
```

Comment:
What is this?

Reply:
It allows setting the number of decimal places in a double (used in framerate display)